### PR TITLE
Support new contains filters

### PR DESCRIPTION
**Pull request by tsmith023**
Adds support for the new `ContainsAny` and `ContainsAll` filters.

These must be used in conjunction with `valueText` and `valueString` as lists of strings. This is not checked within the client itself so will result in a Weaviate REST exception upon user error. This nuance should therefore be communicated to users by DevRel.

When implementing this in the experimental collections API, these checks will be included to aid UX.

### CODE REVIEW

File: weaviate/gql/filter.py [hunk 1]
```diff
@@ -25,6 +25,23 @@
...
+    "ContainsAll",
+    "ContainsAny",
+    "Equal",
+    "GreaterThan",
+    "GreaterThanEqual",
+    "IsNull",
+    "LessThan",
+    "LessThanEqual",
+    "Like",
+    "Not",
```
Comment (dirkkul) [inline weaviate/gql/filter.py:39]: this one is gone

**Review by dirkkul — CHANGES_REQUESTED** [review]
Could you please add an integration test?

File: weaviate/gql/filter.py (hunk 1 above)
Comment (tsmith023) [reply weaviate/gql/filter.py:50]: What do you mean exactly?

Comment (dirkkul) [reply weaviate/gql/filter.py:50]: "Not" does not exist anymore. It never worked in the first place, so we removed it from weaviate

File: integration/test_graphql.py [hunk 2]
```diff
@@ -111,7 +111,44 @@ def test_get_data(client):
...
+def test_get_data_with_where_contains_any(client: weaviate.Client):
+    """Test GraphQL's Get clause with where filter."""
+    where_filter = {"path": ["size"], "operator": "ContainsAny", "valueIntList": [5]}
+    result = client.query.get("Ship", ["name", "size"]).with_where(where_filter).do()
+    objects = get_objects_from_result(result)
+    a_found = False
+    for obj in objects:
+        if obj["name"] == "HMS British Name":
+            a_found = True
+    assert a_found and len(objects) == 1
```
Comment (dirkkul) [inline integration/test_graphql.py:123]: just do
assert len(objects) == 1 and objects[0]["name"] == "HMS..."

File: integration/test_graphql.py [hunk 3]
```diff
@@ -111,7 +111,44 @@ def test_get_data(client):
...
+
+
+def test_get_data_with_where_contains_all(client: weaviate.Client):
```
Comment (dirkkul) [inline integration/test_graphql.py:126]: can you parameterize the test?

**Review by dirkkul — APPROVED** [review]

File: weaviate/gql/filter.py (hunk 1 above)
Comment (tsmith023) [reply weaviate/gql/filter.py:50]: Ahh, okay! Will remove

**Review by dirkkul — APPROVED** [review]

**Pull request merged**